### PR TITLE
Drop 3.7

### PR DESCRIPTION
**Pull request by xiki-tempula**
Drop 3.7 as test is failing 
According to NEP29, we should be dropping py3.8 so I don’t think there is any need to fix 3.7
I have also add 3.11 as the world is moving fast.

### CODE REVIEW

**Review by orbeckst — CHANGES_REQUESTED** [review]
Also change setup.py please. 

I don’t think we have changes to update, have we?

**Comment by xiki-tempula** [comment]
@orbeckst There is one commit. I'm working on fixing this CI as the CI is failing to trigger for https://github.com/alchemistry/alchemtest/pull/87

**Comment by xiki-tempula** [comment]
FIxed #85 as well

File: .github/workflows/CI.yaml [hunk 1]
```diff
@@ -73,8 +73,9 @@ jobs:
         pytest -v --cov=alchemtest --cov-report=xml --color=yes src/alchemtest/tests
 
     - name: CodeCov
-      uses: codecov/codecov-action@v1
+      uses: codecov/codecov-action@v2
```
Comment (orbeckst) [inline .github/workflows/CI.yaml:76]: use v3
```suggestion
      uses: codecov/codecov-action@v3
```

**Review by orbeckst — APPROVED** [review]
lgtm, will just update the codecov action version

**Pull request merged**